src/main/scala/io/iohk/ethereum/consensus/ethash/Miner.scala

```diff
@@ -24,14 +25,14 @@ import scala.util.{Failure, Random, Success, Try}
 import io.iohk.ethereum.utils.BigIntExtensionMethods._
 
 class Miner(
-    blockchain: Blockchain,
-    blockGenerator: BlockGenerator,
-    ommersPool: ActorRef,
-    pendingTransactionsManager: ActorRef,
-    syncController: ActorRef,
-    miningConfig: MiningConfig,
-    ethService: EthService)
-  extends Actor with ActorLogging {
+  blockchain: Blockchain,
+  blockGenerator: BlockGenerator,
+  ommersPool: ActorRef,
+  pendingTransactionsManager: ActorRef,
+  syncController: ActorRef,
+  config: FullConsensusConfig[MiningConfig],
+  ethService: EthService
+) extends Actor with ActorLogging {
```

Comment:
Not a big deal, but the [style guide](https://docs.scala-lang.org/style/declarations.html) recommends params to be indented with 4 spaces.

Reply:
:smile:  That style guide is inconsistent (4 vs 2 spaces) and not even Lightbend uses it, e.g. check [akka](https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/routing/Balancing.scala#L68..L72). I usually indent with 2 spaces, and Twitter does [the same](http://twitter.github.io/effectivescala/#Formatting-Whitespace).

Reply:
But of course, I am happy to adhere to any convention we have, the only reason I followed these patterns is out of habbit and because that I was not aware of any local conventions.

Reply:
I know it's not widely adhered to, but where did you find inconsistency within it?

Anyway it's just a proposal. Feel free to disregard it. I personally like the 4 space rule - it makes it clearer where parameters end. I'm always trying to promote conforming to this guide, but our codebase is already inconsistent in this and other regards.